lua_symbols.ldsym

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
for the final implementation I imagine you can move this to a subdirectory and use a filegroup to include it, and then add it as an additional_linker_input to the rules that must consume it